api/app/src/main/kotlin/packit/model/dto/SubmitRunInfo.kt

```diff
@@ -0,0 +1,8 @@
+package packit.model.dto
+
+data class SubmitRunInfo(
+    val name: String,
```

Comment:
```suggestion
    val packetGroupName: String,
```

Reply:
similarly to the comment above with emma suggesting `commitHash` instead of `hash`, the runner takes `name` so idm doing `packetGroupName` and doing a json property annotation of `name`?

Reply:
makes sense happy to leave as is